services/singleuser/Dockerfile

```diff
@@ -1,12 +1,37 @@
 # Build as renku/singleuser
 # Run with the DockerSpawner in JupyterHub
 
-ARG BASE_IMAGE=renku/minimal-notebook:latest
+ARG JUPYTERHUB_VERSION=0.8.1
+ARG BASE_IMAGE=jupyterhub/singleuser:$JUPYTERHUB_VERSION
+
 FROM $BASE_IMAGE
+ARG JUPYTERHUB_VERSION
```

Comment:
From the Docker docs: "An ARG declared before a FROM is outside of a build stage, so it can’t be used in any instruction after a FROM. To use the default value of an ARG declared before the first FROM use an ARG instruction without a value inside of a build stage."

The first time it is included to set the ARG, the second time to 'import' it to the build stage. If the second one is not included, it won't be recognized further down the line.

Reply:
So unless we want to hard code the initial version, this is the way to do it.

Reply:
ok thanks for clarifying -- I remembered there was something weird about the `ARG` but couldn't recall the details!